src/organisms/forms/GeneralForm/index.js

```diff
@@ -8,19 +8,11 @@ function GeneralForm( props ) {
   const {
     className,
     children,
-    formName,
-    onSubmit,
-
   } = props;
 
   return (
-    <div className={classnames( styles[''], className )}>
-      <form
-        onSubmit={onSubmit}
-        name={formName}
-      >
-        { children }
-      </form>
+    <div className={classnames( styles['general-form'], className )}>
+      { children }
```

Comment:
@patrickkim @drewdrewthis the only benefit is [the css](https://github.com/policygenius/athenaeum/pull/223/files#diff-85c27aa18b6dc3064078588be7297e3dR38), which is adding horizontal padding on small screen sizes in order to accommodate [this design](https://app.zeplin.io/project/58e6498f59f26ca94d348983/screen/590c9b1082a395bab0c9e57e), which requires the individual components to set their own padding instead of the checkout container. I can't inline this because it's a media query, and I don't really want to write a wrapper component because it seems to be more of a unique use case than, for example, the `Hide` component I just wrote. The only other thing I can think of is to introduce a stylesheet to renters -- what are your thoughts?